openapi.yaml

```diff
@@ -0,0 +1,800 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Conquery API
+  description: Conquery API
+  license:
+    name: TODO
+    url: http://TODO/  
+components:
+
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+
+    Ingef:
+      type: http
+      scheme: bearer
+  parameters:
+    dataset:
+      in: path
+      name: dataset
+      required: true
+      schema:
+        $ref: "#/components/schemas/datasetId"
+    query:
+      in: path
+      name: query
+      required: true
+      schema:
+        $ref: "#/components/schemas/executionId"
+    concept:
+      in: path
+      name: concept
+      required: true
+      schema:
+        $ref: "#/components/schemas/conceptId"
+  responses:
+    unauthorized:
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: The user is not logged in.
+  schemas:
+    id:
+      type: string
+    tableId:
+      $ref: "#/components/schemas/id"
+    datasetId:
+      $ref: "#/components/schemas/id"
+    filterId:
+      $ref: "#/components/schemas/id"
+    selectId:
+      $ref: "#/components/schemas/id"
+    conceptId:
+      $ref: "#/components/schemas/id"
+    connectorId:
+      $ref: "#/components/schemas/id"
+    secondaryIdId:
+      $ref: "#/components/schemas/id"
+    columnId:
+      $ref: "#/components/schemas/id"
+    executionId:
+      $ref: "#/components/schemas/id"
+    userId:
+      $ref: "#/components/schemas/id"
+    dateTime: #TODO
+      type: string
+      # format: date-time TODO not compatible: Value does not conform to the required ISO-8601 datetime format. Invalid value '2022-12-13T13:33:32.455901838Z' for type datetime at
```

Comment:
Sollten wir hier auf https://stackoverflow.com/questions/289311/output-rfc-3339-timestamp-in-java umsatteln?

Alternativ das vielleicht das "handy" Pattern am Ende [dieses Posts](https://www.myintervals.com/blog/2009/05/20/iso-8601-date-validation-that-doesnt-suck/) für den ISO-Standard

Reply:
Wenn wir es einfach umstellen können, wäre das nicer, weil die code-generatoren format=date-time verstehen und das schon als dateTime destructuren

Reply:
Man kann das Ausgabe Datumsformat mit `com.fasterxml.jackson.databind.ObjectMapper#setDateFormat` ändern. Das bräuchte etwas Koordination, aber wäre cool, weil es dann eben in den automatisch erzeugten Apis direkt als Datum gelesen werden würde

Reply:
Das wäre cool! Also für das Frontend und so wäre das interessant?